axlearn/cloud/gcp/jobs/tpu_runner.py

```diff
@@ -299,6 +299,21 @@ def _start(self):
         if tier is not None:
             reserved = str(tier) == "0"
             logging.info("Found tier=%s in env. Using reserved=%s", tier, reserved)
+        
+        # Create labels for vm tier that can be used to group tpu metrics.
+        # In QRM, vm tier can be one of guaranteed, spot, or, bestEffort.
+        # bestEffort is implemented to create spot instance. Hence,
+        # guaranteeded -> reserved and bestEffort|spot -> spot 
+        # BASTION_TIER env has presendence over the reserved_tpu
+        labels = {}
+        if reserved is None:
+            reserved_tier = gcp_settings("reserved_tpu", default=False, required=False)
+        else:
+            reserved_tier = reserved 
+        if reserved_tier:
+            labels["vmtier"] = "reserved"
+        else:
+            labels["vmtier"] = "spot"
```

Comment:
```suggestion
        if reserved is None:
            reserved = gcp_settings("reserved_tpu", default=False, required=False)
        labels = {"bastion_tier": "reserved" if reserved else "spot"}
```

Not sure if `vmtier` label is arbitrary?

Reply:
@markblee made the code changes as suggested.  Custom label names are user defined. "bastion_tier" makes more sense as a label name in the context.